### DISCUSSION TITLE
client does not load ltsp image

### DISCUSSION THREAD

**Discussion by Squialz** [ltsp/ltsp#720] [Q&A]
hello finally I opted to make the ltsp server in physics and the client is fine until the ipxe boot menu but after it launches the bone which is on its disk and not the image on the ltsp server (I specify that the image I uploaded to the server is in .vmdk)
![image](https://user-images.githubusercontent.com/103559520/171582972-2a737631-74bf-4b41-9c28-36ea17f2c63f.png)

**Comment by alkisg** [comment]
I'm not sure what you mean, but iPXE can't read the local disk at all.
Try to upload a video of the boot process, it might explain the issue better.

**Comment by Squialz** [comment]
VID_20220602_102039.mp4
<https://drive.google.com/file/d/1njw_ZNne0GW3H22nA8FvftjuBIK2COZe/view?usp=drivesdk>

**Comment by alkisg** [comment]
That link is not publicly accessible.

**Comment by Squialz** [comment]
the link is public now

**Comment by Squialz** [comment]
after the video the pc starts on the image it already has on disk

**Comment by alkisg** [comment]
OK, the error message in that video is:

> /ltsp/ltsp.img... No such file or directory

Which means that you didn't run the [ltsp initrd](https://ltsp.org/man/ltsp-initrd/) command in your server, which generates ltsp.img.

**Comment by Squialz** [comment]
I will write the ltsp initrd command should I delete the ltsp.img file?

**Comment by alkisg** [comment]
The ltsp.img file doesn't exist now, you can't delete something that doesn't exist.
The `ltsp initrd` command will generate it.
The LTSP clients cannot boot without ltsp.img.

**Comment by Squialz** [comment]
perfect it works thank you very much
